rainier-test/src/test/scala/com/stripe/rainier/compute/ComputeTest.scala

```diff
@@ -0,0 +1,16 @@
+package com.stripe.rainier.compute
+
+import org.scalatest._
+
+trait ComputeTest extends FunSuite {
+  def assertWithinEpsilon(x: Double, y: Double, clue: String): Unit = {
+    if (x.abs < 10E-8 && y.abs < 10E-8)
+      // both values really close to 0, consider equal to
+      // account for inaccuracies in numeric differentiation
+      return ()
```

Comment:
I would use `else` rather than `return` here. There is no performance win to using the return and it's not idiomatic (a reviewer has to always check that it isn't inside a closure which has different rules).